src/EnergyPlus/OutputProcessor.cc

```diff
@@ -5550,7 +5547,9 @@ void SetupOutputVariable(std::string const &VariableName,           // String Na
 
     if (!OutputInitialized) InitializeOutput();
 
-    VarName = VariableName;
+    // Variable name without units
+    std::string VarName = VariableName;
+    //  CHARACTER(len=MaxNameLength) :: VariableNamewithUnits ! Variable name with units std format
```

Comment:
You don't need this line.

```suggestion
```

Reply:
Just an FYI, I didn't add this comment, I just move the varName declaration (and the comment) to initialize on the spot rather than at the top of the file.